_install_guide/auth.md

```diff
@@ -90,6 +89,8 @@ security:
       email: email
       firstName: given_name
       lastName: family_name
+    userInfoRequirements: # Used to filter access by userInfo attributes
+      email: /^.*@armory.io$/ # Accepts a Java regular expression
```

Comment:
Well, it accepts a regular expression or an ordinary string match. Either works

Reply:
@avram would it be helpful if I appended `or string to match against.` to the comment?